.github/workflows/ci.yaml

```diff
@@ -750,27 +657,34 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build and push Linux amd64 Docker image
-        id: build_and_push
+      - name: Build
         run: |
-          set -euxo pipefail
+          set -euo pipefail
```

Comment:
Was the `-x` here not providing anything useful?

Reply:
Added back